lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const types = require('../utils/types');
+
+function makeErrorMessageForGetWithDefault(property) {
+  return `Use \`get\` and \`||\` instead of \`getWithDefault\` for property \`${property}\``;
```

Comment:
I don't think we should encourage `get()`.

Reply:
Included "object-method syntax" from https://blog.emberjs.com/2019/02/11/coming-soon-in-ember-octane-part-1.html
>>>
Use object-method syntax or `get` with `||` instead of `getWithDefault` for property `key`

Reply:
But we have a [no-get](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-get.md) rule that is also enabled in the `octane` config which would be conflicting advice.

Reply:
Makes sense. I've changed the warning so it's consistent with no-get